src/main/java/org/sagebionetworks/template/vpc/VpcTemplateBuilderImpl.java

```diff
@@ -83,6 +68,18 @@ public void buildAndDeploy() throws InterruptedException {
 				.withTags(stackTagsProvider.getStackTags())
 				.withParameters(params));
 		this.cloudFormationClient.waitForStackToComplete(stackName);
+		// Wire SES with SNS endpoints on prod
+		setupSesTopics(stackName);
```

Comment:
We could decouple the SNS topics from the VPC into its own deployment step so that we won't have this issue in the future.